styles/styles.css

```diff
@@ -102,6 +102,7 @@ h1 {
 h1 b, h1 strong {
   text-transform: uppercase;
   padding-right: 5px;
+  word-wrap: break-word;
```

Comment:
I'm not sure we should break words arbitrarily via CSS. The meaning of the word may change.

Reply:
OK, good point.